Tensile/KernelWriterAssembly.py

```diff
@@ -8464,14 +8662,18 @@ def setupStoreElementsForBatch(self, kernel, gwvw, batchElements, batchElementSg
 
         if self.cfg.numVgprsPerDataPerVI > 0:
           if self.cfg.halfDataRegPerVI:
-            if elementIdx%2 == 0:
-              # allocate for two elements:
+            if kernel["ProblemType"]["DataType"].isBFloat16():
```

Comment:
Does this change apply to FP16?

Reply:
yes. But we didn't implement FP16 mfma yet.
Anthony will complete FP16 generator later and change it for FP16.